### DISCUSSION TITLE
Supply imagePullSecrets to runner pod

### DISCUSSION THREAD

**Discussion by GarinKartes** [flux-iac/tofu-controller#1178] [Q&A]
I rebuilt the tf-runner to have the required version of the terraform binary we need and pushed this into our private repo.  I see imagePullSecrets in the values file of the helm chart for the controller, but not the runner.  I am getting an access denied when trying to pull the runner image from our private repo since it doesn't accept the value like the controller.  How do I get this to work?

**Comment by chanwit** [comment]
I think we have a generic Pod template implemented in the main branch.
We are in the process of publishing the controller under new name `tofu-controller`. Hope you would be able to try it once the new version got released.
